client/tooltips.js

```diff
@@ -0,0 +1,156 @@
+module.exports = {
+  speciesEditor: {
+    name: "Names for species, parameters, reactions, events, and rules must be unique.",
+
+    initialValue: "Initial population of a species.",
+
+    annotation: "An optional note about the species.",
+
+    remove: "A species may only be removed if it is not a part of any reaction, event assignment, or rule.",
+    
+    speciesMode: "Concentration - Species will only be represented as deterministic.<br>Population - Species will only be represented as stochastic.<br>Hybrid "+
+        "Concentration/Population - Allows a species to be represented as either deterministic or stochastic. This allow you to customize the mode of individual species "+
+        "and set the switching tolerance or minimum value for switching.",
+
+    mode: "Concentration - Species will only be represented as deterministic.<br>Population - Species will only be represented as stochastic.<br>Hybrid "+
```

Comment:
"Concentration - Species will only be represented using continuous (floating point) values"
"Population - Species will only be represented using discrete (integer count) values."
"Concentration/Population - Allows a species to be represented continuous and/or discrete values."